core/base/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# syntax=docker/dockerfile-upstream:1.4.3
+
+# base system image (intermediate)
+ARG DISTRO=alpine:3.14.5
+FROM $DISTRO as system
+
+ENV TZ Etc/UTC
+ENV LANG C.UTF-8
+
+RUN set -euxo pipefail \
+  ; adduser -s /bin/bash -Dh /app -k /var/empty -u 1000 -g mailu app \
```

Comment:
should the UID be configurable?

Reply:
I would say: no. But it could help people syncing their data rootless (like in #2459).
If we make this configurable it should be documented and only be possible via env file.
People changing this should know what they are doing.
It's technically not used yet - I could also remove this line and we add again it when switching to non-root containers.